docs/src/pages/components/switches/ColorSwitches.tsx

```diff
@@ -4,13 +4,13 @@ import { green } from '@material-ui/core/colors';
 import Switch from '@material-ui/core/Switch';
 
 const GreenSwitch = styled(Switch)(({ theme }) => ({
-  '& .MuiSwitch-input.Mui-checked': {
+  '&.MuiSwitch-root > .Mui-checked': {
     color: green[600],
     '&:hover': {
       backgroundColor: alpha(green[600], theme.palette.action.hoverOpacity),
     },
   },
-  '& .MuiSwitch-input.Mui-checked + .MuiSwitch-track': {
+  '&.MuiSwitch-root > .Mui-checked + .MuiSwitch-track': {
     backgroundColor: green[600],
   },
 }));
```

Comment:
I would personally 👍 for not doing this change. The incentive for using `.MuiSwitch-input` was so that we don't have a `.Mui-checked` "floats around" with not obviously host element it comes with.

Reply:
Using `.MuiSwitch-input` custom color doesn't work. https://next.material-ui.com/components/switches/#color

Reply:
Oh, oops, I might have broken it then. It should be `.MuiSwitch-switchBase`.